server/src/main.rs

```diff
@@ -46,6 +51,18 @@ async fn main() -> Result<()> {
             .data(graphql::make_schema(settings.clone(), pool.clone()))
             .wrap(middleware::Compress::default())
             .wrap(middleware::Logger::default())
+            .wrap(IdentityService::new(
+                // <- create identity middleware
+                CookieIdentityPolicy::new(&[0; 32]) // <- create cookie identity policy
```

Comment:
Let's pass in the key via Settings

Reply:
base64 encoded, maybe

Reply:
see below, this will happen in a different PR. Also, I think we probably need .env or something so we don't leak a private key via git.

Reply:
What I've been doing so far with e.g. the database password is to check in an invalid default, with the expectation that those will be passed via the environment.

This works best if we validate the passed-in values before starting the server.

Reply:
ack. closing since its not a part of this PR anymore.